test/easyconfigs/easyconfigs.py

```diff
@@ -1764,6 +1764,20 @@ def template_easyconfig_test(self, spec):
             fail_msg += f"'{orig_val}' vs '{dumped_val}'"
             failing_checks.append(fail_msg)
 
+    # don't allow updating of $PYTHONPATH with standard lib/python*/site-packages path,
+    # since that's already taken care of by EasyBuild framework now
+    # (cfr. https://github.com/easybuilders/easybuild-framework/pull/4539)
+    modextrapaths = ec.get('modextrapaths', {}, resolve=True)
```

Comment:
We usually denote explicitly when we do **not** want to resolve values so I'd not be so explicit (only) here and given `ec.get('key') == `ec['key']`

```suggestion
    modextrapaths = ec.get('modextrapaths', {})
```

Reply:
I prefer making it very explicit here that we're dealing with a resolved value, otherwise the regular expression being used below doesn't make sense (since it doesn't take into account templates like `%(pyshortver)s`).

Reply:
Ok. For me that wouldn't have been a question because we always deal with resolved values when working with an `EasyConfig` instance and usually have to do extra work when we don't want/can't do that. So this felt inconsistent to the other uses of `ec.get` we have.  
But as it did cause confusion, even for someone as familiar with the codebase, it might make sense to be so explicit here.